### PR TITLE
Create simple dropdown component

### PR DESCRIPTION
**Pull request by CalMlynarczyk**
### Description
Create a dropdown component in the vein of the sort selector on https://zpl.io/bz6OY9z.

### Testing Instructions
Storybook

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3060)

### CODE REVIEW

**Comment by dwolfand** [comment]
@matthewheck this probably serves the purpose you were asking about yesterday too!

File: components/atoms/SimpleDropdown/SimpleDropdown.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    return (
+      <Select
+        options={mappedOptions}
+        // value must match the format of options
+        value={{value, label: value}}
+        onChange={this.onChange}
+        isSearchable={false}
+        isClearable={false}
+        components={{DropdownIndicator}}
```
Comment (matthewheck) [inline components/atoms/SimpleDropdown/SimpleDropdown.js]: should this be double {{?

Comment (CalMlynarczyk) [reply components/atoms/SimpleDropdown/SimpleDropdown.js]: Yes, the prop takes an object which is a collection of components you want to overwrite.

Comment (matthewheck) [reply components/atoms/SimpleDropdown/SimpleDropdown.js]: Gotcha. Thanks for explaining!

**Pull request merged**